packages/story-editor/src/components/panels/panel/simplePanel.js

```diff
@@ -31,6 +31,7 @@ function SimplePanel({ children, name, ariaLabel, title, className, ...rest }) {
     <Panel name={name} {...rest}>
       <PanelTitle
         ariaLabel={ariaLabel ?? (typeof title === 'string' ? title : '')}
+        {...rest}
```

Comment:
without this we can't know when `canCollapse` is false in [title](https://github.com/GoogleForCreators/web-stories-wp/pull/10518/files#diff-cb4fb18e75e611c59228354d45b53a41ec97a7a885dba78d148a92d8bebe5f58R166-R175) when `SimplePanel` is used